packages/core/test/render3/BUILD.bazel

```diff
@@ -13,21 +13,30 @@ ts_library(
     tsconfig = "//packages:tsconfig.json",
     deps = [
         "//packages:types",
-        "//packages/core",
-        "//packages/platform-browser",
-        "//packages/platform-browser/animations",
         "//packages/animations",
         "//packages/animations/browser",
         "//packages/animations/browser/testing",
+        "//packages/core",
+        "//packages/platform-browser",
+        "//packages/platform-browser/animations",
     ],
 )
 
 jasmine_node_test(
     name = "render3",
+    srcs = ["load_domino.js"],
```

Comment:
Isn't `load_domino.ts` removed in this PR?

Reply:
that was an accidental deletion